spec/lib/percy/percy_spec.rb

```diff
@@ -0,0 +1,55 @@
+RSpec.describe Percy, type: :feature do
+  TEST_CASE_GLOB =  File.join(File.dirname(__FILE__), "./capybara/client/test_data/test-*.html")
+
+  describe '#snapshot', type: :feature, js: true do
+    context 'with live sites' do
+      it 'snapshots simple HTTPS site' do
+        visit 'https://example.com'
+        Percy.snapshot(page)
+      end
+      it 'snapshots complex HTTPS site' do
+        visit 'https://polaris.shopify.com/'
+        Percy.snapshot(page)
+      end
+      it 'snapshots site with strict CSP' do
+        visit 'https://buildkite.com/'
```

Comment:
Don't love coupling our test implementation to someone else's production config, but non-blocking.